app/backend/src/core/clients/blockchain/IBlockchainClient.ts

```diff
@@ -13,59 +14,31 @@ export interface Block {
   number: number;
   hash: string;
   timestamp: number;
-  parentHash: string;
```

Comment:
These fields are unused and either not present or cumbersome to extract on Starknet.
As such I decided to simply remove them, but I'm open to alternative solutions.